docs/infra/treatment-service.md

```diff
@@ -0,0 +1,132 @@
+# Treatment Service Deployment
+
+## Overview
+XP's Treatment Service is currently offered in two different versions:
+
+- a central Treatment Service, *deployed as a single deployment, to serve treatments to multiple 
+  clients/users with multiple projects*
+- a Treatment Service plugin, *deployed as a Turing Router experiment engine plugin, to serve treatments to a single 
+  client/user with a single project*
+
+## Central Treatment Service
+
+### Requirements
+- Configuration
+- Google Cloud Provider (GCP) service account
+
+### Configuration
+The central Treatment Service deployment uses a variety of configuration that can be stored in one or multiple 
+configuration files (`.yaml`) files. Some of such examples can be found here:
```

Comment:
nit: `Some examples...`

Reply:
nit: variety of `configuration` -> `configurations` ?